hollow/src/test/java/com/netflix/hollow/core/index/HollowHashIndexTest.java

```diff
@@ -56,6 +56,52 @@ public void test() throws Exception {
 
     }
 
+    @Test
+    public void testUpdateListener() throws Exception {
+        HollowObjectMapper mapper = new HollowObjectMapper(writeStateEngine);
+
+        mapper.add(new TypeA(1, 1.1d, new TypeB("one")));
+        mapper.add(new TypeA(1, 1.1d, new TypeB("1")));
+        mapper.add(new TypeA(2, 2.2d, new TypeB("two"), new TypeB("twenty"), new TypeB("two hundred")));
+        mapper.add(new TypeA(3, 3.3d, new TypeB("three"), new TypeB("thirty"), new TypeB("three hundred")));
+
+        roundTripSnapshot();
+
+        HollowHashIndex index = new HollowHashIndex(readStateEngine, "TypeA", "a1", new String[] {"a1", "ab.element.b1.value"});
+        index.listenForDeltaUpdates();
+
+        HollowHashIndexResult result = index.findMatches(2, "twenty");
+        HollowOrdinalIterator iter = result.iterator();
+
+        int matchedOrdinal = iter.next();
+
+        while(matchedOrdinal != HollowOrdinalIterator.NO_MORE_ORDINALS) {
+            System.out.println(matchedOrdinal);
+            matchedOrdinal = iter.next();
+        }
+
+        System.out.println(result.contains(2));
+
+        mapper.add(new TypeA(1, 1.1d, new TypeB("one")));
+        mapper.add(new TypeA(1, 1.1d, new TypeB("1")));
+        mapper.add(new TypeA(2, 2.2d, new TypeB("two"), new TypeB("forty"), new TypeB("two hundred")));
+        mapper.add(new TypeA(3, 3.3d, new TypeB("three"), new TypeB("thirty"), new TypeB("three hundred")));
+        roundTripDelta();
+
+        HollowHashIndexResult result2 = index.findMatches(2, "forty");
+        HollowOrdinalIterator iter2 = result.iterator();
+
+        int matchedOrdinal2 = iter2.next();
+
+        while(matchedOrdinal2 != HollowOrdinalIterator.NO_MORE_ORDINALS) {
+            System.out.println(matchedOrdinal2);
```

Comment:
I think these calls to System.out.println should be Assert statements.  

It appears that the prior test in this class was also written as an exploratory test -- but then the step to convert it to an automated unit test was missed.